src/mono/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.WASI.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<linker>
+	<assembly fullname="System.Private.CoreLib">
+		<!-- these methods are temporarily accessed via UnsafeAccessor from generated code until we have it in public API, probably in WASI preview3 and promises -->
```

Comment:
doesn't UnsafeAccessor preserve the linkage?

Reply:
indeed, it was not necessary. removed